docs/guide/03-styling-templates.md

```diff
@@ -15,6 +15,53 @@ To help with dynamic styling, lit-html provides two directives for manipulating
 *   [`classMap`](template-reference#classmap) sets classes on an element based on the properties of an object.
 *   [`styleMap`](template-reference#stylemap) sets the styles on an element based on a map of style properties and values.
 
+## Inline styles with styleMap {#stylemap}
+
+You can use the `styleMap` directive to set inline styles on an element in the template.
+
+```js
+import {html} from 'lit-html';
+import {styleMap} from 'lit-html/directives/style-map.js';
+
+const normalStyles = {};
+const highlightStyles = { color: 'white', backgroundColor: 'red'};
+let highlight = true;
+
+const myTemplate = () => {
+  html`
+    <div style=${styleMap(highlight ? highlightStyles : normalStyles)}>
+      Hi there!
+    </div>
+  `;
+};
+```
+
+More information: see See [styleMap](template-reference#stylemap) in the Template syntax reference.
+
+## Setting classes with classMap {#classmap}
+
+Like `styleMap`, the `classMap` directive lets you set a group of classes based on an object:
+
+```js
+import {html} from 'lit-html';
+import {classMap} from 'lit-html/directives/class-map.js';
+
+// Define a base set of classes for all menu items
+const baseClasses = { 
+  'menu-item': true,
+  // ...
+};
+
+const itemTemplate = (item) => {
+  // Merge in dynamically-generated classes
+  const mergedClasses = Object.assign({active: item.active}, baseClasses);
```

Comment:
I think the merging might be obscuring the basic functionality here. Maybe just add one class, like `selected`, and remove the `Object.assign`?

Reply:
OK. Like:

```
const classes = {menu-item: true, selected: item.selected}
```

Might be clearer if we expanded it to multi-line:

```
const classes = { 
  // static class
  'menu-item': true,
  // dynamic class
  selected: item.selected,
}
```

... But maybe that's overkill.

Reply:
With `classMap` the static classes don't need to be in the object, they can be part of the HTML, and it's a bit more efficient that way. So I'm not sure about including static classes at all, lest the reader think they have to be put there.

```js
  return html`<div class="menu-item ${classMap(classes)}>Classy text</div>`
```

Reply:
Oh. Huh. Currently our docs say that you _can't_ do that--that classMap, like styleMap, needs to be the entire attribute value. So we need to fix that, too.

https://lit-html.polymer-project.org/guide/template-reference#classmap

Reply:
We say that it must be the only binding, but can be combined with static classes

Reply:
Never mind, reread, says it needs to be the only _binding_, but it can include static content, exactly like this example.